en/services/opendroneid.md

```diff
@@ -46,27 +51,53 @@ Basic ID | Basic ID | [OPEN_DRONE_ID_BASIC_ID](../messages/common.md#OPEN_DRONE_
 Location | Location | [OPEN_DRONE_ID_LOCATION](../messages/common.md#OPEN_DRONE_ID_LOCATION) | Provides location, altitude, direction, and speed of the UA.
 Authentication | Not specified (reserved) | [OPEN_DRONE_ID_AUTHENTICATION](../messages/common.md#OPEN_DRONE_ID_AUTHENTICATION) | Provides authentication data for the UA.
 Self-ID | Self-ID | [OPEN_DRONE_ID_SELF_ID](../messages/common.md#OPEN_DRONE_ID_SELF_ID) | Plain text message that can be used by Operators to identify themselves and the purpose of an operation. Can also be used to provide optional additional clarification in an emergency/remote ID system failure situation.
-System | System | [OPEN_DRONE_ID_SYSTEM](../messages/common.md#OPEN_DRONE_ID_SYSTEM) | Includes Remote Pilot location, multiple aircraft information (group), if applicable, and additional system information.
+<a id="OPEN_DRONE_ID_SYSTEM"></a>System | System | [OPEN_DRONE_ID_SYSTEM](../messages/common.md#OPEN_DRONE_ID_SYSTEM) | Includes Remote Pilot location, multiple aircraft information (group), if applicable, and additional system information.
 Operator ID | Operator ID | [OPEN_DRONE_ID_OPERATOR_ID](../messages/common.md#OPEN_DRONE_ID_OPERATOR_ID) | Provides the Operator ID.
 Message Pack | Message Pack | [OPEN_DRONE_ID_MESSAGE_PACK](../messages/common.md#OPEN_DRONE_ID_MESSAGE_PACK) | A payload mechanism for combining the messages above into a single message pack. Used with Bluetooth Extended Advertising, Wi-Fi NaN and Wi-Fi Beacon.
+- | - | [OPEN_DRONE_ID_ARM_STATUS](../messages/common.md#OPEN_DRONE_ID_ARM_STATUS) | Broadcast once per second from the RID transmitter/receiver components, to indicate their state. Similar to [Heartbeat](#heartbeat), but with additional text information in the error field when they are not ready to arm.
+- | - | [OPEN_DRONE_ID_SYSTEM_UPDATE](../messages/common.md#OPEN_DRONE_ID_SYSTEM_UPDATE) | A subset of the [OPEN_DRONE_ID_SYSTEM](#OPEN_DRONE_ID_SYSTEM) message, containing only fields that must be updated regularly by the GCS. Can optionally be used instead of the full `OPEN_DRONE_ID_SYSTEM` message, to reduce the traffic on the control link.
```

Comment:
I'm not certain the semantics are quite "right" here. What this says is that you can send OPEN_DRONE_ID_SYSTEM_UPDATE  _instead_ of OPEN_DRONE_ID_SYSTEM. 

That is possibly true if none of the information in OPEN_DRONE_ID_SYSTEM that is not in OPEN_DRONE_ID_SYSTEM_UPDATE  is mandatory or desired. 

My assumption is that you want the information in both, but that OPEN_DRONE_ID_SYSTEM_UPDATE  contains the stuff that updates at high rate and for which there is possibly also some compliance requriement? Not sure because you refer to **GCS at the target below**.

Anyway, this should probably be reworded to say that if you use only OPEN_DRONE_ID_SYSTEM it must be sent at high rate for compliance. If you use both messages you can send the more efficient OPEN_DRONE_ID_SYSTEM_UPDATE  at high rate and stream OPEN_DRONE_ID_SYSTEM at a much lower rate.

Also clarify if GCS is the destination.

Reply:
Good points. I have tried to clarify the text.
The GCS is the originator.